dev-requirements.txt

```diff
@@ -1,3 +1,7 @@
 django-debug-toolbar==1.5     # https://django-debug-toolbar.readthedocs.io/en/stable/changes.html
 django-extensions==1.7.4
+ipython==5.1.0
+ipython-genutils==0.1.0
+ipdb==0.10.1
+notebook==4.2.3
```

Comment:
Great to have `ipython` and `ipdb`.

Seems to me that if `ipdb` is bringing in `ipython-genutils` we should let it manage the versioning. Wouldn't omitting `ipython-genutils` prevent possible version number conflicts?

I'd leave out `notebook` until we're actually using it. If someone wants to create doc notebooks as you suggest, starting with a `pip install notebook` won't be a high barrier.